src/main/java/org/folio/qm/config/ApplicationConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.folio.qm.config;
+
+import java.util.Set;
+import java.util.concurrent.ConcurrentSkipListSet;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ApplicationConfig {
+  /**
+   * This bean is used to store tenants which were enabled for this module.
+   * It is needed to clear record creation statuses from database for all tenants.
+   * */
+  @Bean
+  public Set<String> tenants() {
```

Comment:
Could we create a class that will be used for holding all tenants the module is enabled for? Maybe some TenantsHolder with methods that add, remove, and return tenants? In this case, we will have no need to have this bean and comment, it will be more transparent for what it needed.